nuntium/templatetags/nuntium_tags.py

```diff
@@ -35,3 +38,15 @@ def join_with_commas(obj_list):
 @register.assignment_tag(takes_context=True)
 def assignment_url_with_subdomain(context, view, subdomain=UNSET, *args, **kwargs):
     return subdomainsurls(context, view, subdomain, *args, **kwargs)
+
+
+@register.simple_tag(takes_context=True)
+def custom_stylesheet(context):
+    output = ''
+    if hasattr(context['request'], 'subdomain') and context['request'].subdomain is not None:
+        path = os.path.join(context['request'].subdomain, 'css/custom.css')
```

Comment:
I've tried to convince myself that the subdomain can't be manipulated to contain any characters that might cause problems, but that's relatively tricky for me to do, anyway.  To be defensive, I think I'd put in a check that the subdomain only contain a limited range of characters (`[-_a-zA-Z]` perhaps?) and rather an exception otherwise.